nativescript/src/components.module.ts

```diff
@@ -4,16 +4,16 @@ import { NativeScriptModule, NativeScriptFormsModule, NativeScriptHttpModule, Na
 import { Http } from '@angular/http';
 
 // angular
-import { NgModule } from '@angular/core';
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA } from '@angular/core';
 
 // libs
 import { StoreModule } from '@ngrx/store';
 import { EffectsModule } from '@ngrx/effects';
 import { TranslateModule, TranslateLoader, TranslateStaticLoader } from 'ng2-translate';
 
 // app
-import { WindowService, ConsoleService, RouterExtensions } from './app/frameworks/core/index';
-import { NSAppComponent } from './pages/app/app.component';
+import { WindowService, ConsoleService, RouterExtensions, AppService } from './app/frameworks/core/index';
```

Comment:
`
import { 
  WindowService, 
  ConsoleService, 
  RouterExtensions, 
  AppService 
} from './app/frameworks/core/index';
`

For anything with more than three imports, particularly with something with a long path multi-line imports tend to be a bit more readable imo.